src/Seq.Client.Log4Net/Client/Log4Net/LoggingEventFormatter.cs

```diff
@@ -52,59 +51,39 @@ static LoggingEventFormatter()
             };
         }
 
-        static readonly IDictionary<string, string> _levelMap = new Dictionary<string, string>
-        {
-            { "DEBUG", "Debug" },
-            { "INFO", "Information" },
-            { "WARN", "Warning" },
-            { "ERROR", "Error" },
-            { "FATAL", "Fatal" }
-        };
-
```

Comment:
Seq now accepts arbitrary levels, so the new version will send through log4net levels as-is. This will be a breaking change for existing package consumers, but will set this package up for a better experience in the long run.